driver/src/main.rs

```diff
@@ -192,8 +201,9 @@ fn main() {
     .unwrap();
 
     // Set up web3 and contract connection.
-    let contract =
-        StableXContractImpl::new(&web3, options.private_key.clone(), options.network_id).unwrap();
+    let contract = Arc::new(
+        StableXContractImpl::new(&web3, options.private_key.clone(), options.network_id).unwrap(),
+    );
```

Comment:
I see that `Arc` is used as mentioned below in the comments about not cloning the contract, but the clone still occurs below in the same spot. Was this an oversight?

Reply:
We clone the `Arc` pointer here instead of the contract itself. `Arc` is basically a shared reference and the `clone` here is saying give "me a reference"!